integrasjon/rest-klient/src/main/java/no/nav/vedtak/felles/integrasjon/rest/RestClientSupportProdusent.java

```diff
@@ -32,12 +32,10 @@
 
     private final OidcRestClient oidcRestClient;
     private final SystemUserOidcRestClient systemUserOidcRestClient;
-    private final UserAndSystemOidcRestClient userAndSystemOidcRestClient;
 
     public RestClientSupportProdusent() {
```

Comment:
Jeg vile ha lagd en generell rest client basert på Jersey, så kan man hekte på interceptorer som snakker med STS og setter token headere etc på den.

Reply:
Resteasy vs Jersey. Vi bør vel velge 1. Er tiden inne for full konvertering?